packages/shared/providers.yaml

```diff
@@ -61,6 +61,16 @@ airtable:
     proxy:
         base_url: https://api.airtable.com
     docs: https://docs.nango.dev/integrations/all/airtable
+algolia:
+    categories:
+        - other
```

Comment:
Feel like a better category can be used here. `search` is the obvious one but unsure if others would exist in this category.

Reply:
Great I will do that.